### DISCUSSION TITLE
Support Intenationalization for ReactPlay

### DISCUSSION THREAD

**Discussion by atapas** [reactplay/react-play#223] [Ideas]
Opening up this discussion for the Internationalization/Localization support for ReactPlay.

Right now, ReactPlay supports only English. We can support other languages like,

- Spanish
- Chinese
- French
- German
- Bangla
- Is anything else?

What do you think, should we do it?

**Comment by Deepak8717** [comment]
Yes, no harm in that. Although, i think may not be much help unless its really popular

**Reply by atapas** [reply]
I agree with your point. One thing I worry about it, the more the app grows it may be more challenging for us to include as number strings will be huge in number. Also, our code is not written so far in a localization supported way.

What I am thinking is, to include the localization support but support only English now. This way all the strings will be externalized and later we only need to do changes outside of the source code.

**Comment by koustov** [comment]
I would suggest having infrastructure prepared even if we are not doing multiple language translations. We will update the language definition files when we feel so.
